docs/src/guide/installation.md

```diff
@@ -81,6 +81,9 @@ For a less intrusive way to run all scons scripts with Python 3 add this to your
 
 	alias scons="/usr/bin/env python3 $(which scons)"
 
+The latest OpenOCD release v0.10.0 (as of May 2020) is too old for most targets.
+You must to compile the [HEAD version of OpenOCD from source][openocd-source].
```

Comment:
```suggestion
You may need to compile the [HEAD version of OpenOCD from source][openocd-source].
```
~90% of all modm targets work fine with OpenOCD 0.10.
I would like to keep the Installation guide as simple as possible.